recipes/webdavclient2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "webdavclient2" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "797f121c4e381cb57d7f46abef513191da74959d70096e435269d9479a71ee7f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - requests
+    - lxml
+    - argcomplete
+
+test:
+  commands:
+    # script currently broken, issue #4
+    # - wdc --help
+  imports:
+    - webdav2
+    - webdav2.client
+
+about:
+  home: https://github.com/appetito/webdav-client-python-2
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please ask upstream to add the license file to `MANIFEST.in` and please link that issue above in a comment. Thanks. 😄

Reply:
xref: https://github.com/appetito/webdav-client-python-2/issues/6